dipy/align/imaffine.py

```diff
@@ -921,16 +933,22 @@ def _init_optimizer(self, static, moving, transform, params0,
         n = transform.get_number_of_parameters()
         self.nparams = n
 
+        static_masked, moving_masked = static, moving
+        if not static_mask is None:
```

Comment:
here `static_mask` object will not be `None` as we asigned static object to it earlier